examples/h2-other-side-closed-exit-0.cjs

```
@@ -0,0 +1,34 @@
+}));
+
+async function main() {
+  for (let i = 0; i < 100; i++) {
+    try {
+      const r = await request('https://edgeupdates.microsoft.com/api/products');
+      console.log(r.statusCode, r.headers, (await r.body.blob()).size);
+    } catch (err) {
+      // console.error(err);
+      // throw err;
+      if (err.code === 'UND_ERR_SOCKET') {
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)